src/style.css

```diff
@@ -1 +1,27 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 12px;
+  width: 12px;
```

Comment:
Fix size for the stars 
<img width="102" alt="image" src="https://github.com/user-attachments/assets/9073cd30-d52b-4718-9b22-94e980b5cf8b">
```suggestion
  height: 16px;
  width: 16px;
```